docs/narr/sessions.rst

```diff
@@ -71,6 +71,35 @@ using the :meth:`pyramid.config.Configurator.set_session_factory` method.
    security doesn't matter", and you are sure your application has no
    cross-site scripting vulnerabilities.
 
+.. _encrypting_the_session:
+
+Encrypting the Session
+----------------------
+
+Pyramid provides an optional session serializer which encrypts the pickled
+session data using the Blowfish algorithm.  You can use this serializer in
+your :app:`Pyramid` application by using the
+:meth:`pyramid.config.Configurator.set_session_factory` method.
+
+.. code-block:: python
+   :linenos:
+
+   from pyramid.session import BaseCookieSessionFactory
+   from pyramid.session import BlowfishPickleSerializer
+   serializer = BlowfishPickleSerializer('itsaseekreet')
+   my_session_factory = BaseCookieSessionFactory(serializer)
```

Comment:
Nah it is public... I forgot because we initially intended to keep it private and I think it slipped through the cracks and is now public.